PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,4 @@ Fixes #
 - [ ] Changelog is up to date.
 - [ ] Typescript typings for properties are up to date.
 - [ ] Designers approved changes or no approval needed.
+- [ ] Check if peer dependencies of components still work.
```

Comment:
I would suggest "Check if peer dependencies of components has any breaking changes"

Reply:
But this sentence could be true and the component would't have to work...